docs/README.md

```diff
@@ -135,3 +135,22 @@ By default, we just display the object name. To override, use the `displayText`
 ### Navigation
 
 If you are adding a new page or want to update the navigation in the sidebar, update the `docs/content/_navigation.json` file.
+
+### Generating and Updating Screenshots
+
```

Comment:
also requires a 'pip install selenium'

Reply:
Other stuff I had to do:
download chromedriver from here https://chromedriver.chromium.org/downloads
Add it to /usr/local/bin
Run `xattr -d com.apple.quarantine /usr/local/bin/chromedriver` (gross)

Reply:
Hmm I feel like how I did this was less gross, but now it's escaping me